stacks/core/dev/stage-1.yml

```diff
@@ -128,3 +152,4 @@ volumes:
   mainflux-things-db-volume:
   mainflux-users-db-volume:
   mainflux-authn-db-volume:
+  mongo-data:
```

Comment:
If [this suggestion]() is accepted, update this also :smile:

Reply:
Updated!